index.html

```diff
@@ -1,67 +1,218 @@
+                                <li class="waks3"><a href="#"></a><img src="image/rubby2.png" alt="Ruby"></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <button class="button2">See Project</button>
+                </div>
+    
+                <div class="works6">
+                    <div class="image-back">
+                        <div class="professional">Profesional Art Printing Data</div>
+                        <div class="paragraphworks">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</div>
+                        <div>
+                            <ul class="boxes2">
+                                <li class="waks1"><a href="#"></a><img src="image/html2.png" alt="HTML"></li>
+                                <li class="waks2"><a href="#"></a><img src="image/bootstrap2.png" alt="Bootstrap"></li>
+                                <li class="waks3"><a href="#"></a><img src="image/rubby2.png" alt="Ruby"></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <button class="button2">See Project</button>
+                </div>
+    
+                <div class="works7">
+                    <div class="image-back">
+                        <div class="professional">Profesional Art Printing Data</div>
+                        <div class="paragraphworks">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</div>
+                        <div>
+                            <ul class="boxes2">
+                                <li class="waks1"><a href="#"></a><img src="image/html2.png" alt="HTML"></li>
+                                <li class="waks2"><a href="#"></a><img src="image/bootstrap2.png" alt="Bootstrap"></li>
+                                <li class="waks3"><a href="#"></a><img src="image/rubby2.png" alt="Ruby"></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <button class="button2">See Project</button>
+                </div>
+            </div>
         </div>
 
-        <div class="group">
-            <img src="image/Group.png" alt="Group">
+        <div class="afterworks">
+            <div class="immediate-about">
+                <div class="aboutme">About Me</div>
+                <div class="aboutpara">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</div>
+                <button class="aboutbutton"></button>
+            </div>
+            <div class="group2"></div>
+            <div class="central-line"><img src="image/centeralignment.png" alt="Central Line"></div>
```

Comment:
Great job so far! I have two small suggestions to make this part of your project even closer to the Figma template:
- [ ] Check the order of this section: The button should be placed below the paragraph, not above it.
- [ ] Check the styles for your button: Make sure to remove the border and add the correct **hover, selected, and active effects**. Additionally, check that the `font-size` of the text matches the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-27&t=ZJcVC8XUqegdTgGs-0).
- [ ] Check the background image that is supposed to be in this section.
- [ ] Ensure that the line between this section and your cards below needs to be centered.

### Your design:
![image](https://user-images.githubusercontent.com/107952724/235326633-b6e9c33a-bb58-4be9-a63e-190ec0c46393.png)

### Figma design:
![image](https://user-images.githubusercontent.com/107952724/235326737-35821ba4-6415-4425-9eba-5b6addea87ef.png)